readme.md

```diff
@@ -1,7 +1,7 @@
 # Hello world
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_hello-world/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_hello-world/report/html_report/)
+- [DEMO LINK](Bohdan-Kovalenko1992)
+- [TEST REPORT LINK](https://github.com/Bohdan-Kovalenko1992/layout_hello-world.git)
```

Comment:
U need to change your name only (https://**Bohdan-Kovalenko1992**.github.io/layout_hello-world/)